react/src/App.jsx

```diff
@@ -0,0 +1,302 @@
+import { useState, useCallback, useEffect } from "react";
+
+function App() {
+  const [position, setPosition] = useState("bottom-right");
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const [selectedPrivacyOption, setSelectedPrivacyOption] = useState("disable");
+  const [selectedFullScreenOption, setSelectedFullScreenOption] =
+    useState("exitFullScreen");
+
+  const [selectedLauncherOption, setSelectedLauncherOption] =
+    useState("showLauncher");
+
+  const [unreadMessageCount, setMessageUnreadCount] = useState(0);
+  useState(false);
+
+  const [message, setMessage] = useState("");
+  const [messageEventListenerIsAdded, setmessageEventListenerIsAdded] =
+    useState(false);
+
+  const onUpdateHelpshiftConfig = () => {
+    window.Helpshift("updateHelpshiftConfig");
+  };
```

Comment:
This is not an event handler. It is just a helper function. It doesn't need to be prefixed with `on`.
Also, just use `window.Helpshift("updateHelpshiftConfig");` directly. 

Creating a wrapper function for a one-line call is not necessary, and it might confuse developer who are looking at this repo for examples of when to run this command.